mungegithub/mungers/release-note-label.go

```diff
@@ -144,20 +144,15 @@ func (r *ReleaseNoteLabel) Munge(obj *github.MungeObject) {
 		return
 	}
 
+	releaseNoteLabelNeededAdded := false
 	if !obj.HasLabel(releaseNoteLabelNeeded) {
 		obj.AddLabel(releaseNoteLabelNeeded)
+		releaseNoteLabelNeededAdded = true
 	}
 
-	if !obj.HasLabel(lgtmLabel) {
-		return
+	if releaseNoteLabelNeededAdded && obj.HasLabel(lgtmLabel) {
+		obj.WriteComment(releaseNoteBody)
 	}
-
-	if obj.HasLabel(doNotMergeLabel) {
-		return
-	}
-
-	obj.WriteComment(releaseNoteBody)
-	obj.AddLabel(doNotMergeLabel)
```

Comment:
Yes but this variable is used only when there was no `release-not-label-needed` yet and then we're adding label `release-note-label-needed` (1st call) - then we want to add comment.
Next call (2nd and others) won't add comment because there will be `release-note-label-needed` already comment won't be added again, see:
```
	releaseNoteLabelNeededAdded := false
	if !obj.HasLabel(releaseNoteLabelNeeded) {
		obj.AddLabel(releaseNoteLabelNeeded)
		releaseNoteLabelNeededAdded = true
	}

	if releaseNoteLabelNeededAdded && obj.HasLabel(lgtmLabel) {
		obj.WriteComment(releaseNoteBody)
	}
```